lib/elm-app-worker.js

```diff
@@ -1,13 +1,21 @@
 /**
- * @import {MessagePort} from 'worker_threads';
+ * @import {MessagePort} from 'node:worker_threads';
  * @import {Ports} from './types/app';
+ * @import {WorkerData} from './types/elm-app-wrapper';
  * @import {SendPort} from './types/promisify-port';
+ * @import {WorkerThreads} from './types/worker';
  */
-const {parentPort, workerData} = require('node:worker_threads');
+
+const workerThreads = require('node:worker_threads');
 const ElmCommunication = require('./elm-communication');
 const loadCompiledElmApp = require('./load-compiled-app');
 const ResultCache = require('./result-cache');
 
+/**
+ * @type {WorkerThreads<WorkerData>}
+ */
```

Comment:
Should the type annotation not ideally be placed at the definition of `workerThreads`? (That at least sounds a lot more intuitive to me)

Reply:
I tried that, but it doesn't like it.